tools/clang/test/CodeGenSPIRV/constant-ps.hlsl2spv

```diff
@@ -0,0 +1,31 @@
+// Run: %dxc -T ps_6_0 -E main
+float4 main(): SV_TARGET
+{
+  return float4(1.0f, 1.0f, 1.0f, 1.0f);
+}
+
+// CHECK-WHOLE-SPIR-V:
+// ; SPIR-V
+// ; Version: 1.0
+// ; Generator: Google spiregg; 0
+// ; Bound: 11
+// ; Schema: 0
+// OpCapability Shader
+// OpMemoryModel Logical GLSL450
+// OpEntryPoint Fragment %main "main" %4
+// OpExecutionMode %main OriginUpperLeft
+// OpName %main "main"
+// OpDecorate %4 Location 0
+// %float = OpTypeFloat 32
+// %v4float = OpTypeVector %float 4
+// %_ptr_Output_v4float = OpTypePointer Output %v4float
+// %void = OpTypeVoid
+// %6 = OpTypeFunction %void
+// %float_0x1pn149 = OpConstant %float 0x1p-149
```

Comment:
This is not the correct value for float `1.0`. It is actually the `uint32_t` value of `1` bitcasting to float.

For more about hexadecimal floating points: http://www.exploringbinary.com/hexadecimal-floating-point-constants/